docs/src/main/sphinx/installation/try-it.rst

```diff
@@ -0,0 +1,265 @@
+Try Trino with Docker
+======================
+
+This guide focuses on your first ten minutes
+using Trino, and relies on Docker to simplify the installation and
+ensure that the CLI runs on your platform of choice.
+
+This Docker container includes both the Trino server and the 
+Trino command line interface (CLI), which provides a terminal-based,
```

Comment:
For this firs use of "CLI' on the page, make it a cross-ref to the CLI doc page:

Reply:
Overall, should this be called the CLI or Interactive CLI and do we need to make any type of distinction?